src/ggrc/assets/mustache/controls/info.mustache

```diff
@@ -17,7 +17,8 @@
       {{>'/static/mustache/base_objects/description.mustache'}}
       {{>'/static/mustache/base_objects/test_plan.mustache'}}
       {{>'/static/mustache/base_objects/notes.mustache'}}
-      {{>'/static/mustache/controls/contacts.mustache'}}
```

Comment:
Please don't remove contacts just yet. We'll remove this once we do the migration that moves contacts into custom roles.